CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+
+## [0.9.4-beta] - 2016-04-20
+### Added
+- Deploy to App Engine flexible environment tools menu item. ([#635](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/635))
+- Support for Maven based projects as deployment sources for App Engine flexible environment deployments ([#600](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/600))
+
+### Changed
+- App Engine flexible environment deployment can be cancelled by disconnecting to our App Engine applicaiton server. ([#581](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/581))
```

Comment:
Also noting that if we leave it up to contributors to do this it would be nice to add the instruction in our project contributors file too.